include/logging/printk.h

```diff
@@ -42,50 +42,80 @@ enum log_level {
-#define pr_warning_str_msg(arg1, arg2) printk_str_msg(WARNING, arg1, arg2);
-
-#define pr_error(arg1) printk(ERR, arg1);
-#define pr_error_bool_msg(arg1, arg2) printk_bool_msg(ERR, arg1, arg2);
-#define pr_error_char(arg1) printk_char(ERROR, arg1);
-#define pr_error_float(arg1) printk_float(ERR, arg1);
-#define pr_error_float_msg(arg1, arg2) printk_float_msg(ERR, arg1, arg2);
-#define pr_error_int(arg1) printk_int(ERR, arg1);
-#define pr_error_int_msg(arg1, arg2) printk_int_msg(ERR, arg1, arg2);
-#define pr_error_str_msg(arg1, arg2) printk_str_msg(ERR, arg1, arg2);
+#define pr_info_bool_msg(arg1, arg2) printk_bool_msg(INFO, arg1, arg2)
+#define pr_info_char(arg1) printk_char(INFO, arg1)
+#define pr_info_float(arg1) printk_float(INFO, arg1)
+#define pr_info_float_msg(arg1, arg2) printk_float_msg(INFO, arg1, arg2)
+#define pr_info_int(arg1) printk_int(INFO, arg1)
+#define pr_info_int_msg(arg1, arg2) printk_int_msg(INFO, arg1, arg2)
+#define pr_info_str_msg(arg1, arg2) printk_str_msg(INFO, arg1, arg2)
+#else // ! DEBUG_BUILD
+#define pr_trace(arg1)	{}
+#define pr_trace_bool_msg(arg1, arg2)	{}
+#define pr_trace_char(arg1)	{}
+#define pr_trace_float(arg1)	{}
+#define pr_trace_float_msg(arg1, arg2)	{}
+#define pr_trace_int(arg1)	{}
+#define pr_trace_int_msg(arg1, arg2)	{}
+#define pr_trace_str_msg(arg1, arg2)	{}
+
+#define pr_debug(arg1) {}
+#define pr_debug_bool_msg(arg1, arg2) {}
+#define pr_debug_char(arg1) {}
+#define pr_debug_float(arg1) {}
+#define pr_debug_float_msg(arg1, arg2) {}
+#define pr_debug_int(arg1) {}
+#define pr_debug_int_msg(arg1, arg2) {}
+#define pr_debug_str_msg(arg1, arg2) {}
+
+#define pr_info(arg1) {}
+#define pr_info_bool_msg(arg1, arg2) {}
+#define pr_info_char(arg1) {}
+#define pr_info_float(arg1) {}
+#define pr_info_float_msg(arg1, arg2) {}
+#define pr_info_int(arg1) {}
+#define pr_info_int_msg(arg1, arg2) {}
+#define pr_info_str_msg(arg1, arg2) {}
+#endif // DEBUG_BUILD
+
+#define pr_warning(arg1) printk(WARNING, arg1)
+#define pr_warning_bool_msg(arg1, arg2) printk_bool_msg(WARNING, arg1, arg2)
+#define pr_warning_char(arg1) printk_char(WARNING, arg1)
+#define pr_warning_float(arg1) printk_float(WARNING, arg1)
+#define pr_warning_float_msg(arg1, arg2) printk_float_msg(WARNING, arg1, arg2)
+#define pr_warning_int(arg1) printk_int(WARNING, arg1)
+#define pr_warning_int_msg(arg1, arg2) printk_int_msg(WARNING, arg1, arg2)
+#define pr_warning_str_msg(arg1, arg2) printk_str_msg(WARNING, arg1, arg2)
+
+#define pr_error(arg1) printk(ERR, arg1)
+#define pr_error_bool_msg(arg1, arg2) printk_bool_msg(ERR, arg1, arg2)
+#define pr_error_char(arg1) printk_char(ERROR, arg1)
+#define pr_error_float(arg1) printk_float(ERR, arg1)
+#define pr_error_float_msg(arg1, arg2) printk_float_msg(ERR, arg1, arg2)
+#define pr_error_int(arg1) printk_int(ERR, arg1)
+#define pr_error_int_msg(arg1, arg2) printk_int_msg(ERR, arg1, arg2)
+#define pr_error_str_msg(arg1, arg2) printk_str_msg(ERR, arg1, arg2)
 
```

Comment:
This change removes pr_info, pr_debug, pr_trace messages in release builds.  They were an attempt to decrease the volume of messages.  I will likely revert these.